docs/metricset-api.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[metricset-api]]
+=== Metricsets
```

Comment:
Hmm are metrics included in the sections about event types? Should be?

Reply:
They are not, mostly because I'm not quite sure I fully understand they are yet.